extensions/wikia/EditPageLayout/js/plugins/Sizechangedevent.js

```diff
@@ -9,7 +9,7 @@
 	WE.plugins.sizechangedevent = $.createClass(WE.plugin,{
 
 		initDom: function() {
-			var self = this, editor = this.editor;
+					var self = this, editor = this.editor;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove the declaration of the unused 'editor' variable. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AUnusedVariable)

[project-id]: wikia:app
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this useless assignment to local variable "editor" [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AS1854)

[project-id]: wikia:app